articles/migrations/guides/extensibility-node12.md

```diff
@@ -188,4 +188,4 @@ or, if the module must be pinned to a specific version:
 
 ### Notes for Node 10 and Node 12 changes
 
-For additional information, please consult Node.js's [Node 10 release notes](https://nodejs.org/fr/blog/release/v10.0.0/) and [Introducing Node.js 12](https://medium.com/@nodejs/introducing-node-js-12-76c41a1b3f3f).
\ No newline at end of file
+For additional information, please consult Node.js's [Node 10 release notes](https://nodejs.org/fr/blog/release/v10.0.0/) and [Introducing Node.js 12](https://medium.com/@nodejs/introducing-node-js-12-76c41a1b3f3f).
```

Comment:
Why are we talking about Node 10 here?

Reply:
Oh, I think it was so people could see everything that's changed in between Node 8 -> Node 12.